client/src/document/index.tsx

```diff
@@ -251,7 +252,10 @@ export function Document(props /* TODO: define a TS interface for this */) {
             </React.Suspense>
           )}
           <article className="main-page-content" lang={doc.locale}>
-            <h1>{doc.title}</h1>
+            <header>
+              <h1>{doc.title}</h1>
+              <BaselineIndicator status={doc.baseline} />
```

Comment:
🟠 If we check for `doc.baseline`, we can make it non-optional in the `BaseLineIndicator()`, and avoid rendering that component (even empty) if `doc.baseline` is not set:

```suggestion
              {doc.baseline && <BaselineIndicator status={doc.baseline} />}
```